sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -690,6 +690,13 @@ class MapboxMap internal constructor(
    *
    * @return An array of rendered features.
    */
+  @Deprecated(
+    "Should be replaced overloaded function taking RenderedQueryGeometry and returning cancelable",
+    replaceWith = ReplaceWith(
+      "queryRenderedFeatures(RenderedQueryGeometry(shape), options, callback).let { cancelable -> }",
+      imports = arrayOf("com.mapbox.maps.RenderedQueryGeometry")
+    ),
+  )
```

Comment:
Had to duplicate annotations here - otherwise overriden methods ReplaceWith is not recognized with Android Studio and doesn't work.

Reply:
Not sure if it's supposed behaviour of the annotation, will file an issue to the Kotlin bug tracker

Reply:
https://youtrack.jetbrains.com/issue/IDEA-283066 @mapbox/maps-android pls add some pluses to push the possible fix

Reply:
Isn't it the default behaviour for ReplaceWith expressions?

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-replace-with/expression.html

Reply:
Sorry, read the ticket again and it seems I misunderstood the problem, and yeah, I agree it's a valid issue to raise.